x-pack/plugins/siem/public/resolver/view/defs.tsx

```diff
@@ -401,3 +358,63 @@ export const SymbolDefinitions = styled(SymbolDefinitionsComponent)`
   width: 0;
   height: 0;
 `;
+
```

Comment:
:reminder_ribbon: One other thing above that has really nothing to do with this PR, but seeing this made me remember. That `SymbolDefinitionsComponent` export should have some kind of logic in/around it that only makes it show up once per document (preferably at the highest level). This wouldn't be a problem as long as you have no more than 1 Resolver at a time open, but it could be a problem if you had 2+. Going to log an issue to that effect and reference this comment.

Reply:
Yea, makes sense. Thanks!